src/main/java/seedu/address/model/session/RecurringSession.java

```diff
@@ -103,7 +103,7 @@ public boolean endBefore(SessionDate sessionDate) {
     /**
      * Returns true if recurring session date starts after {@code SessionDate} date.
      */
-    private boolean startAfter(SessionDate sessionDate) {
+    protected boolean startAfter(SessionDate sessionDate) {
```

Comment:
it's so that i can access it for the test. but now that you mention maybe we only needed public method's tests

Reply:
I think stubs aren't supposed to be free of logic implementation, just free of methods we have no dependency on. I too am not sure but was thinking its probably how it should be since it's similar to the ab3's/existing ways of addstudentTest

Reply:
i think in delete session test we've also simlarly effectively made a "stub" when we 'setup()'

Reply:
Right, maybe I phrased it wrongly. I commented one area where it does not seem to be free of dependencies.

Reply:
righti think you're right about having dependencies